README.md

```diff
@@ -4,40 +4,57 @@
 
 > One paragraph statement about the project.
```

Comment:
**Optional Changes**
- [ ] Let us try and replace this placeholder with a description of what our project is about.